specification/templates/.github/CODEOWNERS

```diff
@@ -4,3 +4,14 @@
 * @signalfx/gdi-TEAM-maintainers @signalfx/gdi-TEAM-approvers
 
 .github/CODEOWNERS @signalfx/gdi-TEAM-maintainers
+
+#####################################################
+#
+# Docs reviewers
+#
+#####################################################
+
+*.md @signalfx/docs
+*.rst @signalfx/docs
+docs/ @signalfx/docs
+README* @signalfx/docs @signalfx/gdi-specification-approvers @signalfx/gdi-specification-maintainers
```

Comment:
That will create PRs that do not have `@signalfx/gdi-TEAM-maintainers @signalfx/gdi-TEAM-approvers` as reviewers. Is that really correct? I think that it's great for the docs team to review those files, but the repo maintainers should also be added - to ensure that the docs are correct and match the actual code in the repo.

Reply:
@mateuszrzeszutek can you open a PR to fix this?

Reply:
https://github.com/signalfx/gdi-specification/pull/117